package.json

```diff
@@ -17,17 +18,19 @@
     "publish": "git pull origin master --rebase && npm run build && npm test && lerna publish",
     "publish-internal": "git pull origin master --rebase && npm run build && NPM_DIST_TAG=internal ./node_modules/.bin/lerna publish --skip-git",
     "test": "jest",
-    "test-ci": "npm run bootstrap && npm run build && npm run coverage",
+    "test-ci": "npm run bootstrap && npm run build && npm run build-standalone && npm run coverage",
     "updated": "lerna updated",
     "watch": "gulp watch"
   },
   "devDependencies": {
-    "babel-core": "^6.25.0",
+    "babel-core": "^7.0.0-alpha.19",
```

Comment:
Is this an important/required change for including babel-minify-standalone ?

If so, then all the plugin and preset tests will run on babel7 instead of babel6, and almost everyone using this minifier would be using it with babel6.

Reply:
I had some build errors with 6.x. I can try again and see if I can get it working. I think it was because the latest `babel-standalone` versions reference babel-core 7.x.

Is there a branch of babel-minify for Babel 7.x builds?

Reply:
There is a open PR for babel 7 upgrade here https://github.com/babel/minify/pull/487